### ISSUE TITLE
Validation error: Invalid locale 'true'

### ISSUE THREAD

**Issue by ketysek** [sequelize/sequelize#5407]
Hi, I've got a problem with validation. In my angular fullstack user test it fails with the following error:

```
#password
      1) "before each" hook for "should authenticate user if valid"

  171 passing (404ms)
  1 failing

  1) User Model #password "before each" hook for "should authenticate user if valid":
     SequelizeValidationError: Validation error: Invalid locale 'true'
      at node_modules/sequelize/lib/instance-validator.js:74:14
      at bound (domain.js:287:14)
      at runBound (domain.js:300:12)
      at tryCatcher (node_modules/bluebird/js/release/util.js:16:23)
      at Promise._settlePromiseFromHandler (node_modules/bluebird/js/release/promise.js:497:31)
      at Promise._settlePromise (node_modules/bluebird/js/release/promise.js:554:18)
      at Promise._settlePromise0 (node_modules/bluebird/js/release/promise.js:599:10)
      at Promise._settlePromises (node_modules/bluebird/js/release/promise.js:682:18)
      at Promise._fulfill (node_modules/bluebird/js/release/promise.js:623:18)
      at PromiseArray._resolve (node_modules/bluebird/js/release/promise_array.js:125:19)
      at PromiseArray._promiseFulfilled (node_modules/bluebird/js/release/promise_array.js:143:14)
      at Promise._settlePromise (node_modules/bluebird/js/release/promise.js:559:26)
      at Promise._settlePromise0 (node_modules/bluebird/js/release/promise.js:599:10)
      at Promise._settlePromises (node_modules/bluebird/js/release/promise.js:682:18)
      at Async._drainQueue (node_modules/bluebird/js/release/async.js:125:16)
      at Async._drainQueues (node_modules/bluebird/js/release/async.js:135:10)

Warning: Task "mochaTest:unit" failed. Use --force to continue.
```

My user class is here:
```
var User = sequelize.define('User', {

    _id: {
      type: DataTypes.INTEGER,
      allowNull: false,
      primaryKey: true,
      autoIncrement: true
    },
    avatarPath: DataTypes.STRING,
    name: {
      type: DataTypes.STRING,
      validate: {
        isAlphanumeric: true
      }
    },
    surname: {
      type: DataTypes.STRING,
      validate: {
        isAlphanumeric: true
      }
    },
    nickname: {
      type: DataTypes.STRING,
      unique: true,
      validate: {
        isAlphanumeric: true
      }
    },
    email: {
      type: DataTypes.STRING,
      unique: {
        msg: 'The specified email address is already in use.'
      },
      validate: {
        isEmail: true
      }
    },
    role: {
      type: DataTypes.ENUM('user', 'media', 'admin'),
      defaultValue: 'user'
    },
    userRank: {
      type: DataTypes.ENUM('registered', 'verified', 'expert', 'guru'),
      defaultValue: 'registered'
    },
    ipAddress: DataTypes.STRING,
    blocked: {
      type: DataTypes.BOOLEAN,
      defaultValue: false
    },
    password: {
      type: DataTypes.STRING,
      validate: {
        notEmpty: true
      }
    },
    provider: DataTypes.STRING,
    salt: DataTypes.STRING,
    facebook: DataTypes.TEXT,
    twitter: DataTypes.TEXT,
    google: DataTypes.TEXT,
    github: DataTypes.TEXT,
  }
```

What's wrong? I cannot figure it out :(

**Comment by Subastkar** [comment]
That issue started with us yesterday, after moving around stuff we found out that there is an issue with instance-validator where the package is not supporting numeric validations any more.

```
validator deprecated you tried to validate a number but this library (validator.js) validates strings only.
Please update your code as this will be an error soon. node_modules/sequelize/lib/instance-validator.js:274:33
```

I'm trying to find if this is possible to fix it on my code or maybe I should freeze the dependencies until they fix this issue.

**Comment by Subastkar** [comment]
Looks they already had the answer https://github.com/sequelize/sequelize/issues/5377

**Comment by mickhansen** [comment]
Known issue, need to fix the validation code in Sequelize

**Comment by mickhansen** [comment]
It shouldn't fail though, only log an error

**Comment by detailyang** [comment]
+1 :+1:  got Validation error: Invalid locale \'true\' too in the latest version :(

**Comment by janmeier** [comment]
@detailyang Try lastest master

**Comment by detailyang** [comment]
my sequelize is sequelize  "sequelize": "^3.19.2" and use isAlphanumeric get Validation error: Invalid locale 'true'  :alien:

**Comment by janmeier** [comment]
@detailyang Thats because latest master is not yet published to npm - https://github.com/sequelize/sequelize/pull/5429

**Comment by detailyang** [comment]
@janmeier thx, i know the reason now :grin:

**Comment by kjhangiani** [comment]
just came across this error, still having issues using the commit ref'd at #5429.  Is this fixed in master or in 3.19.3? 

```
validator deprecated you tried to validate a number but this library (validator.js) validates strings only. Please update your code as this will be an error soon. at node_modules/sequelize/lib/instance-validator.js:273:33
```

Was using 3.19.0, and also tried commit eb064697bf53331fec0049a272c1548ceb890600

The model in question has validations defined at the model level, but all fairly simple - just 3 of them, all like this:

```
{
	min: {
		args: [1],
		msg: "You must provide an Entity"
	}
}
```
